cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -294,6 +328,10 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
         endif()
         set(MFEM_USE_UMPIRE OFF CACHE BOOL "")
         set(MFEM_USE_ZLIB ON CACHE BOOL "")
+        if(ENZYME_DIR)
+            serac_assert_is_directory(DIR_VARIABLE ENZYME_DIR)
+            set(MFEM_USE_ENZYME ON CACHE BOOL "")
```

Comment:
in case `MFEM_USE_ENZYME`'s default changes

```suggestion
            set(MFEM_USE_ENZYME ON CACHE BOOL "")
        else()
            set(MFEM_USE_ENZYME OFF CACHE BOOL "")
```